### PR TITLE
MNT Run all behat suites

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/.github/issues/275

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -26,3 +26,102 @@ jobs:
         - php: 8.2
           phpunit: true
           phpunit_suite: recipe-cms
+        - php: 8.3
+          phpunit: true
+          phpunit_suite: recipe-cms
```
Comment (ChloeSartorelli) [inline .github/workflows/ci.yml:31]:
```suggestion
```

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (emteknetnz) [reply .github/workflows/ci.yml:31]: Seems like we should keep it here - we support [php 8.3 in 5.2](https://www.silverstripe.org/blog/cms-5-2/) - we currently have an 8.3 test in the `5` [branch of sink](https://github.com/silverstripe/recipe-kitchen-sink/blob/5/.github/workflows/ci.yml#L29), so may as well backport that to 5.2 as well

Comment (ChloeSartorelli) [reply .github/workflows/ci.yml:31]: Ahh, I thought 8.3 was in 5.3. This is fine then.

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**